docs/src/user-docs/guides-using-presto.md

```
@@ -198,6 +198,43 @@ SELECT foo.bar FROM default LIMIT 1;
 CLP's UI should be available at [http://localhost:4000](http://localhost:4000) (if you changed
 `webui.host` or `webui.port` in `etc/clp-config.yml`, use the new values).
 
+[Figure 1](#figure-1) shows the search page after applying a filter.
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)